docs/keto/configuration.md

```diff
@@ -3,82 +3,92 @@ id: configuration
 title: Configuration
 ---
 
-```yaml
 # ORY Keto Configuration
-#
-#
+
+You can configure ORY Keto via a config file or environment variables.

```

Comment:
This was all in comments in the config file. I have extracted the parts that don't seem related to configuring Keto, like how to set up env variables or Docker, into Markdown documentation on top of this file.

Reply:
This is being auto-generated from [here](https://github.com/ory/keto/blob/master/docs/config.yaml) and thus overwritten on every git push to master. The script doing that is this one: https://github.com/ory/keto/blob/master/scripts/run-configuration.sh

Not a merge blocker, but changes will be lost on next push to keto.